code/modules/research/xenobiology/xenobio_camera.dm

```diff
@@ -25,10 +25,41 @@
 	var/list/stored_slimes = list()
 	var/max_slimes = 5
 	var/monkeys = 0
+	var/mob/living/carbon/human/monkey_type = /mob/living/carbon/human/monkey
+	var/monkey_name = "monkey" //because even abusing initial() isn't enough to get the species proper name, based on how monkey/species setting works.
+	var/cycle_through = 0
 
 	icon_screen = "slime_comp"
 	icon_keyboard = "rd_key"
 
+/obj/machinery/computer/camera_advanced/xenobio/attackby(obj/item/O, mob/user)
+	if(istype(O, /obj/item/device/multitool))
+		cycle_through++
+		switch(cycle_through)
+			if(1)
+				monkey_type = /mob/living/carbon/human/farwa
+				monkey_name = "farwa"
+			if(2)
+				monkey_type = /mob/living/carbon/human/wolpin
+				monkey_name = "wolpin"
+			if(3)
+				monkey_type = /mob/living/carbon/human/stok
+				monkey_name = "stok"
+			if(4)
+				monkey_type = /mob/living/carbon/human/neara
+				monkey_name = "neara"
+			if(5)
+				monkey_type = /mob/living/carbon/human/monkey
+				monkey_name = "monkey"
+				cycle_through = 0
+		to_chat(user, "<span class='notice'>You change the monkeycube type to [monkey_name].</span>")
+	else
```

Comment:
@Fox-McCloud this should be based on species identifier then than trying to find name
```
for(var/datum/species/S in subtypesof(/datum/species)
    if(initial(S.is_small) == 1)
        monkey_species += initial(S.name)